list_of_prime_numbers/main.py

```diff
@@ -1,11 +1,21 @@
 
 def _is_prime(number):
-    pass
+    if number<2:                        #numbers equal to 1 and below are not prime numbers
+        return False
+    if number%2==0 and number>2:        #all even numbers except 2 are not prime numbers
+        return False
+    for i in range (3,number,2):        #checks if number is divisible by any number between 3 and number. 2 steps to skip even numbers. This can be improved with int(n**0.5) + 1 which I have not figured out yet. 
+            if number%i == 0:
+                return False
+    return True                         #loop ends here if above conditions not met. We have a PRIME number!
 
```

Comment:
Food for thought: Apart from tackling this by finding out what **is** a prime number, maybe it will be even more efficient if we can think about what **is not** a prime number...